### PR TITLE
add docstrings to tf classes

### PR DESCRIPTION
**Pull request by Ghostvv**
**Proposed changes**:
- add docstrings to tf classes
- part of https://github.com/RasaHQ/rasa/issues/5319

**Status (please check what you already did)**:
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/master/changelog) for instructions)
- [x] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

File: rasa/utils/tensorflow/layers.py [hunk 1]
```diff
@@ -34,7 +57,47 @@ def dropped_inputs() -> tf.Tensor:
...
+    `output = activation(dot(input, kernel) + bias)`
+    where `activation` is the element-wise activation function
+    passed as the `activation` argument, `kernel` is a weights matrix
+    created by the layer, and `bias` is a bias vector created by the layer
+    (only applicable if `use_bias` is `True`).
+
+    Note: If the input to the layer has a rank greater than 2, then
+    it is flattened prior to the initial dot product with `kernel`.
+
+    Arguments:
```
Comment (tabergma) [inline rasa/utils/tensorflow/layers.py]: You are missing `reg_lambda` here.

File: rasa/utils/tensorflow/layers.py [hunk 2]
```diff
@@ -10,9 +10,32 @@
...
+        """Apply dropout to sparse inputs.
+
+        Arguments:
+            inputs: Input sparse tensor (of any rank).
+            training: Python boolean indicating whether the layer should behave in
+                training mode (adding dropout) or in inference mode (doing nothing).
+
+        Returns:
+            Output of dropout layer.
+        """
```
Comment (tabergma) [inline rasa/utils/tensorflow/layers.py]: Strictly speaking, you also need to add a `Raises` to the doc string if the function raises an exception...

File: rasa/utils/tensorflow/layers.py [hunk 3]
```diff
@@ -34,7 +57,47 @@ def dropped_inputs() -> tf.Tensor:
...
+
+    Arguments:
+        units: Positive integer, dimensionality of the output space.
+            activation: Activation function to use.
+            If you don't specify anything, no activation is applied
+            (ie. "linear" activation: `a(x) = x`).
+        use_bias: Boolean, whether the layer uses a bias vector.
+        kernel_initializer: Initializer for the `kernel` weights matrix.
+        bias_initializer: Initializer for the bias vector.
+        kernel_regularizer: Regularizer function applied to
```
Comment (tabergma) [inline rasa/utils/tensorflow/layers.py]: This one is created in `init`, users cannot pass it as an argument.

File: rasa/utils/tensorflow/layers.py [hunk 4]
```diff
@@ -160,6 +311,19 @@ def call(self, x: tf.Tensor) -> tf.Tensor:
 
 
 class InputMask(tf.keras.layers.Layer):
```
Comment (tabergma) [inline rasa/utils/tensorflow/layers.py]: Should we add a doc string to the `call` method.

**Comment by Ghostvv** [comment]
how the coverage managed to decrease?

**Comment by tabergma** [comment]
Not sure how the check is working but could be simple because more lines were added to the code. So number of lines covered by tests divided by total number of lines decreased. But not sure about that.

**Pull request merged**